samples/boot/oauth2/src/main/java/org/springframework/security/samples/config/Message.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
```

Comment:
Looks good. The only one change is the samples directory structure. Currently the sample lives in `samples/boot/oauth2`, however, it should go into `samples/boot/oauth2/resource-server`.

For example, the `authorization_code` code grant sample (pending PR) has a sample in `samples/boot/oauth2/authcodegrant`. 

I'm also wondering if it would be better under `samples/boot/oauth2/client/authcodegrant` and your sample under `samples/boot/oauth2/resource-server/jwt-provider`?